test/api/v3/integration/groups/GET-groups.test.js

```diff
@@ -0,0 +1,101 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
+
+describe('GET /group', () => {
+  let user;
+
+  beforeEach(async () => {
+    user = await generateUser({balance: 4});
+  });
+
+  it('returns an error when group is not found', async () => {
+    await expect(user.get('/groups/fakeGroupId', {}))
```

Comment:
This file is for testing `GET /groups` and this test is testing `GET /groups/:id`